### ISSUE TITLE
Empty list when running `gpo_member_photos.py` script

### ISSUE THREAD

**Issue by mdrayer** [unitedstates/images#221]
It seems that the scraper is no longer working. I get a "Just a moment..." HTML page for the `response` (https://github.com/unitedstates/images/blob/gh-pages/scripts/gpo_member_photos.py#L54). This can be replicated by doing a `curl` on the URL the scraper is trying to hit: https://www.congress.gov/search?q=%7B%22source%22%3A+%22members%22%2C+%22congress%22%3A+%22118%22%7D&pageSize=250&page=1.

Are there any troubleshooting techniques here to make the scraper wait a bit for the HTML to load + form?

**Comment by hugovk** [comment]
Yeah, I think it's been a few years since the scraper script has been run to fetch images, so it's unsurprising it's not working properly. Scrapers needs quite a bit of upkeep!

Unfortunately I don't really have time to maintain this repo any more, so we should find some new maintainers willing to volunteer.

**Comment by mdrayer** [comment]
GPO.gov _does_ have an API that includes these images we are attempting to scrape: https://pictorialapi.gpo.gov/. There's an endpoint to get all members of a particular session, e.g. 118th Congress: https://pictorialapi.gpo.gov/api/GuideMember/GetMembers/118 which yields images like https://memberguide.gpo.gov/pictorialimages/118_SR_NJ_Booker_Cory.jpg. However, it does _not_ include the member's bioguide ID in the data, so we'd need to do some mapping with another dataset to get the bioguide ID.

**Comment by kevinschaul** [comment]
FYI I'm working to get pictorial ids into the legislators project: https://github.com/unitedstates/congress-legislators/pull/943

**Comment by kevinschaul** [comment]
Update: Pictorial ids are in! Next step would be to use those to download the images as @mdrayer suggested. Not sure when I'll get around to that but it is on my list.

**Comment by kevinschaul** [comment]
Another update to this. Unfortunately pictorial does not include all members. For example they are missing Tony Wied: https://www.congress.gov/member/tony-wied/W000829

I'm now thinking that updating the Congress.gov scraper is the better option.

**Comment by kevinschaul** [comment]
^ I've got this system working but am not 100 percent sure whether this runs into copyright issues [raised here](https://github.com/sunlightlabs/congress/issues/432#issuecomment-34481338) (and referenced in this repo's README.md), since these are not from GPO.

Also the Pictorial images seem generally newer and better. I'm going to focus my efforts there instead.
